frontend/src/app/shared/data/languageLevel.ts

```diff
@@ -1,6 +1,6 @@
 export enum LanguageLevel {
-    B1 = 'B1',
-    B2 = 'B2',
-    C1 = 'C1',
-    C2 = 'C2',
+    B1 = 'B1 (Intermediate)',
+    B2 = 'B2 (Upper-Intermediate)',
+    C1 = 'C1 (Advanced)',
+    C2 = 'C2 (Proficient)',
```

Comment:
it would be better to have mapper for this
Enam should contains only raw data (the same as in the backend)